src/pages/filter/ui/panels/function-panel/components/compound-request/control-buttons.tsx

```diff
@@ -0,0 +1,50 @@
+import { ReactElement } from 'react'
+import cn from 'classnames'
+import { observer } from 'mobx-react-lite'
+
+import { Button } from '@ui/button'
+import compoundRequestStore from './compound-request.store'
+
+interface IControlButtonsProps {
+  activeRequestIndex: number
+  setActiveRequestIndex: (item: number) => void
+}
+
+export const ControlButtons = observer(
+  ({
+    activeRequestIndex,
+    setActiveRequestIndex,
+  }: IControlButtonsProps): ReactElement => (
+    <div className="flex">
+      <Button
+        onClick={() =>
+          compoundRequestStore.handleRequestBlocksAmount(
+            'ADD',
```

Comment:
May be we can use enum for `ADD` and `REMOVE` values?

Reply:
sure